component/pxctl/pxc-pxctl

```diff
@@ -32,7 +32,10 @@ if [ "$1" = "-n" ] ; then
 	NODE=$2
 	shift 2
 else
-	nodes=$(kubectl pxc node list -o json | jq -r '.[].hostname')
+	k8snodes=$(kubectl get pod -n portworx -lname=portworx -o jsonpath='{.items[*].spec.nodeName}')
```

Comment:
This command assumes `portworx` namespace. Is it possible to pass in the namespace instead?